webapp/platform/client/src/websocket.ts

```diff
@@ -412,6 +412,18 @@ export default class WebSocketClient {
         this.sendMessage('user_update_active_status', data, callback);
     }
 
+    postedNotifyAck(postId: string, result: 'error' | 'not_sent' | 'unsupported' | 'success', reason?: string, postedData?: string) {
```

Comment:
Could this method be renamed to something more self-explanatory like `acknowledgePostedNotification`? I know what `postedNotifyAck` means from context, but I feel like people are going to have no idea what this does in the future based on the name.

`usePostedAckHook` on the server is also pretty arcane as well, but that's at least not part of a public package

Reply:
Sure, changed